.github/workflows/e2e_tests_cypress.yml

```diff
@@ -28,7 +26,17 @@ jobs:
         # Ignore the failure of a step and avoid terminating the job.
         continue-on-error: true
 
-      - run: cd tests && ./run-docker up --build -d
+      - name: Build and start docker images
+        run: cd tests && ./run-docker up --build -d
+
+      - name: Wait docker images ready and install Lizmap
+        run: cd tests && sleep 30 && ./lizmap-ctl install
```

Comment:
Just in case, it took me some times to know how to do multiple command with **multiple** lines, not on a single line : 

```yml
    - name: Wait docker images ready and install Lizmap
        run:  |
          cd tests
          sleep 30
          ./lizmap-ctl install
```

The `|` operator. `>` also exists in YML.

My 2 cents ;-)